src/NuGet.Services.EndToEnd/Support/Utilities/PortableMetadataReader.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Reflection.Metadata;
+
+namespace NuGet.Services.EndToEnd.Support.Utilities
+{
+    public static class PortableMetadataReader
+    {
+        public static string GetIndex(string pdbFullPath)
+        {
+            var indexingPath = string.Empty;
+            if (pdbFullPath == null)
+            {
+                throw new ArgumentNullException(nameof(pdbFullPath));
+            }
+
+            using (var stream = File.OpenRead(pdbFullPath))
+            using (var pdbReaderProvider = MetadataReaderProvider.FromPortablePdbStream(stream, MetadataStreamOptions.LeaveOpen))
+            {
+                var pdbReader = pdbReaderProvider.GetMetadataReader();
+                var pdbSignature = new BlobContentId(pdbReader.DebugMetadataHeader.Id).Guid;
+                var pdbFileName = Path.GetFileName(pdbFullPath).ToLowerInvariant();
+                var pdbAge = "FFFFFFFF";
```

Comment:
>FFFFFFFF [](start = 30, length = 8)

This should be "ffffffff" I believe

Reply:
Here's the latest implementation on BaGet: [https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.Core/Services/SymbolIndexingService.cs#L151](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet.Core/Services/SymbolIndexingService.cs#L151)

I forget offhand what was broken with the uppercasing

---
In reply to: [241852502](https://github.com/NuGet/NuGet.Services.EndToEnd/pull/65#discussion_r241852502) [](ancestors = 241852502)

Reply:
I don't think the casing matters here. I tried out all the casings and it worked fine when hitting the symbol server endpoint.